### PR TITLE
Fixing the use case "Establish OpenShift cluster upgrade process"

### PR DESCRIPTION
**Pull request by computate**
Because there is no custom documentation for NERC OpenShift upgrades, The team decided that the following step did not apply to acceptance tests:
 "Upgrade documentation available in the documentation repository".
This commit updates the acceptance tests to point to the official OpenShift cluster updates documentation instead:
https://docs.openshift.com/container-platform/4.10/updating/index.html

### CODE REVIEW

File: acceptance-tests.md
Comment (larsks) [inline acceptance-tests.md]: Remove `here` in the link and a minor grammar fix:

```suggestion
        1. See the official [OpenShift updating clusters documentation](https://docs.openshift.com/container-platform/4.10/updating/index.html) for the version of OpenShift to which you wish to upgrade.
```

**Review by larsks — DISMISSED** [review]

File: acceptance-tests.md
Comment (computate) [reply acceptance-tests.md]: @larsks Thanks for the suggestion, I have made this change and fixed quotes.

**Comment by joachimweyl** [comment]
> Because there is no custom documentation for NERC OpenShift upgrades, The team decided that the following step did not apply to acceptance tests: "Upgrade documentation available in the documentation repository". This commit updates the acceptance tests to point to the official OpenShift cluster updates documentation instead: https://docs.openshift.com/container-platform/4.10/updating/index.html

Hello @waygil & @syockel,
Please review the notes from the NERC meeting you were unable to attend to ensure that you agree with the decision made.

**Review by hpdempsey — APPROVED** [review]
Approved.  Note to Scott and Wayne from Kim is to make sure they notice.  No need to wait for them before merging.

**Comment by syockel** [comment]
Okay just stating that I caught up on this thread.

**Comment by waygil** [comment]
Reviewed and agree.

**Review by hpdempsey — APPROVED** [review]
Changes look good and Scott and Wayne have agreed.  Ready to merge.

**Comment by computate** [comment]
@larsks could you please review this one again? It's waiting for your approval.

**Comment by hpdempsey** [comment]
> @larsks could you please review this one again? It's waiting for your approval.

This is a very simple change Chris, I think we can skip the second review
from Lars, especially because he already was involved in the conversation
about resolving it.

**Pull request merged**
